.github/workflows/verify-pr-commit.yml

```diff
@@ -119,20 +128,21 @@ jobs:
         run: |
           CARGO_INCREMENTAL=0 RUSTFLAGS="-D warnings" cargo build --locked --release \
             --features ${{matrix.spec}}
+      - name: Rename Reference Binary
+        if: steps.cache-binary.outputs.cache-hit != 'true'
+        working-directory: ${{env.BIN_DIR}}
+        run: cp ${{env.BUILT_BIN_FILENAME}} ${{env.FINAL_BIN_FILENAME}}
```

Comment:
Is it worth doing the rename? Couldn't we just use `env.BUILT_BIN_FILENAME` everywhere we currently use `env.FINAL_BIN_FILENAME`?